Source/JavaScriptCore/unexported-libc++.txt

```diff
@@ -0,0 +1,34 @@
+# Some libc++ declarations declare symbol visibility, such that even though WTF and JavaScriptCore build with
+# -fvisibilty=hidden, the std:: symbols are exported. This unexport list uses the Itanium mangling rules [1]
+# to attempt to hide all such symbols from the JavaScriptCore executable. As these symbols all come from
+# headers, symbols that slip through are weak and will be caught by the "Check For Weak VTables and Externals"
+# build script phase.
```

Comment:
I think that the explanation here is a little confusing. AFAICT it's exporting these symbols that's the real problem (perhaps especially new/delete), not that a verifier complains.

But this makes me wonder why libc++ forces symbol visibility. Is it necessary for it in some way?

Super excited about this change though.